packages/cli-repl/test/repl-helpers.ts

```diff
@@ -77,29 +77,40 @@ async function readReplLogfile(logPath: string) {
 }
 
 
-const fakeExternalEditor = async(output?: string) => {
-  const base = path.resolve(__dirname, '..', '..', '..', 'tmp', 'test', `${Date.now()}`, `${Math.random()}`);
-  const tmpDoc = path.join(base, 'editor-script.js');
+const fakeExternalEditor = async(
+  { output, expectedExtension, tmpdir }: { output?: string, expectedExtension?: string, tmpdir: string }
+) => {
+  const editor = (expectedExtension === '.mongodb') ? 'code' : 'editor';
```

Comment:
I liked the approach where the filename is passed explicitly that you had before – this looks somewhat surprising, because it’s not obvious from the test itself that this is what makes the editor have `code` as its filename. And additionally, this would break a test where we’d test that we still use `.js` for `code` when the extension is not installed (which we should have).